packages/gitgraph-node/src/compute-graph-map.test.ts

```diff
@@ -0,0 +1,329 @@
+import { GitgraphCore, Branch, Template } from "gitgraph-core/lib";
```

Comment:
Why do you need to import gitgraph-core here? We are testing gitgraph-node here so we should only import from gitgraph-node

Reply:
BTW, we should be able to use `gitgraph-core` directly (without the `/lib`) I know that I've done the same hack into gitgraph-react but it's a hack ^^

Reply:
For the record:
- we didn't manage to get rid of `/lib`, yet
- yes, we are testing `gitgraph-node` here, but we need to use types defined in `gitgraph-core/lib` and we set up some custom template to test `computeGraphMap(gitgraph)`